chapter_06/rkc/exercise_1.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+# Our knowledge base
+my %person = (fred => 'flintstone', barney => 'rubble', wilma => 'flintstone');
+
+# Takes a first name and returns the person's surname or undef if not found
+sub lookup_surname {
+	my ($name) = @_;
+	return $person{$name};
+}
```

Comment:
Moving the hash lookup to a function is maybe a little more than is needed here, although I can see it being useful if you were to change the data storage method.